openfisca_france/parameters/taxation_capital/epargne/livret_epargne_populaire/taux.yaml

```
@@ -4,6 +4,8 @@ values:
     value: 0.06
   2024-02-01:
     value: 0.05
+  2024-08-01:
+    value: 0.04
 metadata:
   short_label: Taux
```

Je propose d'ajouter le champ `last_value_still_valid_on`, même si il est égal à la date de la nouvelle valeur.
```suggestion
  last_value_still_valid_on: "2024-08-01"
  short_label: Taux
```

---

Aurais-tu plus de billes / une documentation sur l'impact et le fonctionnement de cette metadata ? Je ne suis pas très familier avec l'environnement et j'ai du mal à trouver des sources à ce sujet

---

Bonjour, oui il y a un explication dans le wiki : https://github.com/openfisca/openfisca-france/wiki/Bonnes-pratiques-de-mod%C3%A9lisation#last_value_still_valid_on

---

Ah merci beaucoup pour le lien 🙏